ecommerce/extensions/payment/processors/stripe.py

```diff
@@ -62,6 +64,8 @@ def __init__(self, site):
         self.publishable_key = configuration['publishable_key']
         # The secret API key used by the backend to communicate with Stripe. Private/secret.
         self.secret_key = configuration['secret_key']
+        # The webhook endpoint secret used by Stripe to secure the endpoint . Private/secret.
+        self.endpoint_secret = configuration['endpoint_secret']
```

Comment:
In light of new PR https://github.com/openedx/ecommerce/pull/3908, don't forget to:

```suggestion
        self.endpoint_secret = configuration['webhook_endpoint_secret']
```

(I think the rest will be taken care of by a rebase.)

Reply:
I was hesitant to make so many changes in the other PR so that I could pin point the fix, however, ended up adding everything from this commit except for that.

I'll add it in and remove this commit from this PR, thanks!